[2024-01-02T19:56:59.013+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T19:56:59.028+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T19:56:59.029+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T19:56:59.097+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T19:56:59.104+0000] {standard_task_runner.py:57} INFO - Started process 3866 to run task
[2024-01-02T19:56:59.109+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmp0g4m3rty']
[2024-01-02T19:56:59.112+0000] {standard_task_runner.py:85} INFO - Job 16: Subtask transform_data
[2024-01-02T19:56:59.199+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host ce0bd5f35f4e
[2024-01-02T19:56:59.607+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T19:56:59.642+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:47 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T19:56:59.647+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 221, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_with_postgress_operator2.py", line 47, in transform_data
    data.booking_date = pd.to_datetime(data.booking_date, infer_datetime_format=True)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/tools/datetimes.py", line 1050, in to_datetime
    values = convert_listlike(arg._values, format)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/tools/datetimes.py", line 453, in _convert_listlike_datetimes
    return _array_strptime_with_fallback(arg, name, utc, format, exact, errors)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/tools/datetimes.py", line 484, in _array_strptime_with_fallback
    result, timezones = array_strptime(arg, fmt, exact=exact, errors=errors, utc=utc)
  File "pandas/_libs/tslibs/strptime.pyx", line 530, in pandas._libs.tslibs.strptime.array_strptime
  File "pandas/_libs/tslibs/strptime.pyx", line 351, in pandas._libs.tslibs.strptime.array_strptime
ValueError: time data "2017-10-17" doesn't match format "%Y/%m/%d", at position 1. You might want to try:
    - passing `format` if your strings have a consistent format;
    - passing `format='ISO8601'` if your strings are all ISO8601 but not necessarily in exactly the same format;
    - passing `format='mixed'`, and the format will be inferred for each element individually. You might want to use `dayfirst` alongside this.
[2024-01-02T19:56:59.671+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T195659, end_date=20240102T195659
[2024-01-02T19:56:59.712+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 16 for task transform_data (time data "2017-10-17" doesn't match format "%Y/%m/%d", at position 1. You might want to try:
    - passing `format` if your strings have a consistent format;
    - passing `format='ISO8601'` if your strings are all ISO8601 but not necessarily in exactly the same format;
    - passing `format='mixed'`, and the format will be inferred for each element individually. You might want to use `dayfirst` alongside this.; 3866)
[2024-01-02T19:56:59.764+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-01-02T19:56:59.850+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-01-02T19:59:42.140+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T19:59:42.161+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T19:59:42.162+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T19:59:42.218+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T19:59:42.225+0000] {standard_task_runner.py:57} INFO - Started process 3971 to run task
[2024-01-02T19:59:42.230+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmpy4jteo_c']
[2024-01-02T19:59:42.233+0000] {standard_task_runner.py:85} INFO - Job 17: Subtask transform_data
[2024-01-02T19:59:42.327+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host ce0bd5f35f4e
[2024-01-02T19:59:42.669+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T19:59:42.712+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:47 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T19:59:42.722+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 221, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_with_postgress_operator2.py", line 47, in transform_data
    data.booking_date = pd.to_datetime(data.booking_date, infer_datetime_format=True)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/tools/datetimes.py", line 1050, in to_datetime
    values = convert_listlike(arg._values, format)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/tools/datetimes.py", line 453, in _convert_listlike_datetimes
    return _array_strptime_with_fallback(arg, name, utc, format, exact, errors)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/tools/datetimes.py", line 484, in _array_strptime_with_fallback
    result, timezones = array_strptime(arg, fmt, exact=exact, errors=errors, utc=utc)
  File "pandas/_libs/tslibs/strptime.pyx", line 530, in pandas._libs.tslibs.strptime.array_strptime
  File "pandas/_libs/tslibs/strptime.pyx", line 351, in pandas._libs.tslibs.strptime.array_strptime
ValueError: time data "2018/03/20" doesn't match format "%Y-%m-%d", at position 7. You might want to try:
    - passing `format` if your strings have a consistent format;
    - passing `format='ISO8601'` if your strings are all ISO8601 but not necessarily in exactly the same format;
    - passing `format='mixed'`, and the format will be inferred for each element individually. You might want to use `dayfirst` alongside this.
[2024-01-02T19:59:42.737+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T195942, end_date=20240102T195942
[2024-01-02T19:59:42.794+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 17 for task transform_data (time data "2018/03/20" doesn't match format "%Y-%m-%d", at position 7. You might want to try:
    - passing `format` if your strings have a consistent format;
    - passing `format='ISO8601'` if your strings are all ISO8601 but not necessarily in exactly the same format;
    - passing `format='mixed'`, and the format will be inferred for each element individually. You might want to use `dayfirst` alongside this.; 3971)
[2024-01-02T19:59:42.844+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-01-02T19:59:42.884+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-01-02T20:00:47.485+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T20:00:47.499+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T20:00:47.500+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T20:00:47.560+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T20:00:47.569+0000] {standard_task_runner.py:57} INFO - Started process 4023 to run task
[2024-01-02T20:00:47.573+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmprp7_4ilp']
[2024-01-02T20:00:47.578+0000] {standard_task_runner.py:85} INFO - Job 18: Subtask transform_data
[2024-01-02T20:00:47.675+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host ce0bd5f35f4e
[2024-01-02T20:00:48.076+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T20:00:48.124+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:47 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T20:00:48.187+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 221, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_with_postgress_operator2.py", line 54, in transform_data
    data = data.drop('address', 1)
TypeError: drop() takes from 1 to 2 positional arguments but 3 were given
[2024-01-02T20:00:48.205+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T200047, end_date=20240102T200048
[2024-01-02T20:00:48.258+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 18 for task transform_data (drop() takes from 1 to 2 positional arguments but 3 were given; 4023)
[2024-01-02T20:00:48.309+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-01-02T20:00:48.347+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-01-02T20:03:08.784+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T20:03:08.837+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T20:03:08.838+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T20:03:08.911+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T20:03:08.921+0000] {standard_task_runner.py:57} INFO - Started process 4152 to run task
[2024-01-02T20:03:08.927+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmph7_lx8ey']
[2024-01-02T20:03:08.932+0000] {standard_task_runner.py:85} INFO - Job 19: Subtask transform_data
[2024-01-02T20:03:09.093+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host ce0bd5f35f4e
[2024-01-02T20:03:09.494+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T20:03:09.530+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:47 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T20:03:09.540+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 221, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_with_postgress_operator2.py", line 54, in transform_data
    data = data.drop('address', 1)
TypeError: drop() takes from 1 to 2 positional arguments but 3 were given
[2024-01-02T20:03:09.555+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T200308, end_date=20240102T200309
[2024-01-02T20:03:09.586+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 19 for task transform_data (drop() takes from 1 to 2 positional arguments but 3 were given; 4152)
[2024-01-02T20:03:09.623+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-01-02T20:03:09.656+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-01-02T20:06:10.414+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T20:06:10.430+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T20:06:10.430+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T20:06:10.527+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T20:06:10.537+0000] {standard_task_runner.py:57} INFO - Started process 4299 to run task
[2024-01-02T20:06:10.544+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmpq953iubb']
[2024-01-02T20:06:10.548+0000] {standard_task_runner.py:85} INFO - Job 20: Subtask transform_data
[2024-01-02T20:06:10.679+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host ce0bd5f35f4e
[2024-01-02T20:06:11.413+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T20:06:11.458+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:47 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T20:06:11.473+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 221, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_with_postgress_operator2.py", line 54, in transform_data
    data = data.drop('address', 1)
TypeError: drop() takes from 1 to 2 positional arguments but 3 were given
[2024-01-02T20:06:11.490+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T200610, end_date=20240102T200611
[2024-01-02T20:06:11.558+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 20 for task transform_data (drop() takes from 1 to 2 positional arguments but 3 were given; 4299)
[2024-01-02T20:06:11.601+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-01-02T20:06:11.637+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-01-02T20:09:37.062+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T20:09:37.074+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T20:09:37.075+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T20:09:37.112+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T20:09:37.118+0000] {standard_task_runner.py:57} INFO - Started process 4446 to run task
[2024-01-02T20:09:37.123+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmpcx7to60s']
[2024-01-02T20:09:37.126+0000] {standard_task_runner.py:85} INFO - Job 21: Subtask transform_data
[2024-01-02T20:09:37.237+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host ce0bd5f35f4e
[2024-01-02T20:09:37.747+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T20:09:37.783+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:47 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T20:09:37.794+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 221, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_with_postgress_operator2.py", line 54, in transform_data
    data = data.drop('address', 1)
TypeError: drop() takes from 1 to 2 positional arguments but 3 were given
[2024-01-02T20:09:37.810+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T200937, end_date=20240102T200937
[2024-01-02T20:09:37.856+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 21 for task transform_data (drop() takes from 1 to 2 positional arguments but 3 were given; 4446)
[2024-01-02T20:09:37.898+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-01-02T20:09:37.939+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-01-02T20:35:01.102+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T20:35:01.120+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T20:35:01.121+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T20:35:01.165+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T20:35:01.171+0000] {standard_task_runner.py:57} INFO - Started process 5527 to run task
[2024-01-02T20:35:01.177+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmp5ut9vj9k']
[2024-01-02T20:35:01.180+0000] {standard_task_runner.py:85} INFO - Job 23: Subtask transform_data
[2024-01-02T20:35:01.257+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host ce0bd5f35f4e
[2024-01-02T20:35:01.606+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T20:35:01.643+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:46 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T20:35:01.653+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 221, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_with_postgress_operator2.py", line 53, in transform_data
    data = data.drop(['address'], 1)
TypeError: drop() takes from 1 to 2 positional arguments but 3 were given
[2024-01-02T20:35:01.668+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T203501, end_date=20240102T203501
[2024-01-02T20:35:01.741+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 23 for task transform_data (drop() takes from 1 to 2 positional arguments but 3 were given; 5527)
[2024-01-02T20:35:01.791+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-01-02T20:35:01.824+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-01-02T20:37:25.218+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T20:37:25.232+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T20:37:25.233+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T20:37:25.273+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T20:37:25.280+0000] {standard_task_runner.py:57} INFO - Started process 5638 to run task
[2024-01-02T20:37:25.284+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmpuzx0h1m9']
[2024-01-02T20:37:25.288+0000] {standard_task_runner.py:85} INFO - Job 24: Subtask transform_data
[2024-01-02T20:37:25.361+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host ce0bd5f35f4e
[2024-01-02T20:37:25.757+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T20:37:25.800+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:46 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T20:37:25.864+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 221, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_with_postgress_operator2.py", line 56, in transform_data
    data.to_csv(f"{dag_path}/processed_data/processed_data.csv", index=False)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/generic.py", line 3772, in to_csv
    return DataFrameRenderer(formatter).to_csv(
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/formats/format.py", line 1186, in to_csv
    csv_formatter.save()
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/formats/csvs.py", line 240, in save
    with get_handle(
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/common.py", line 737, in get_handle
    check_parent_directory(str(handle))
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/common.py", line 600, in check_parent_directory
    raise OSError(rf"Cannot save file into a non-existent directory: '{parent}'")
OSError: Cannot save file into a non-existent directory: '/opt/***/processed_data'
[2024-01-02T20:37:26.139+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T203725, end_date=20240102T203726
[2024-01-02T20:37:26.180+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 24 for task transform_data (Cannot save file into a non-existent directory: '/opt/***/processed_data'; 5638)
[2024-01-02T20:37:26.221+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-01-02T20:37:26.265+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-01-02T20:56:58.961+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T20:56:58.978+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T20:56:58.979+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T20:56:59.042+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T20:56:59.054+0000] {standard_task_runner.py:57} INFO - Started process 420 to run task
[2024-01-02T20:56:59.062+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmpol4fuaml']
[2024-01-02T20:56:59.069+0000] {standard_task_runner.py:85} INFO - Job 27: Subtask transform_data
[2024-01-02T20:56:59.209+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host 2590655a4ba7
[2024-01-02T20:56:59.805+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T20:56:59.973+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:46 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T20:57:00.113+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-02T20:57:00.149+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T205658, end_date=20240102T205700
[2024-01-02T20:57:00.441+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-02T20:57:00.514+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-01-02T21:03:59.838+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T21:03:59.870+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T21:03:59.872+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T21:03:59.920+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T21:03:59.932+0000] {standard_task_runner.py:57} INFO - Started process 704 to run task
[2024-01-02T21:03:59.941+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmp3hf94pvf']
[2024-01-02T21:03:59.947+0000] {standard_task_runner.py:85} INFO - Job 29: Subtask transform_data
[2024-01-02T21:04:00.093+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host 2590655a4ba7
[2024-01-02T21:04:00.447+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T21:04:00.947+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:46 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T21:04:01.061+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-02T21:04:01.166+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T210359, end_date=20240102T210401
[2024-01-02T21:04:01.282+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-02T21:04:01.357+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-01-02T21:10:17.366+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T21:10:17.383+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T21:10:17.384+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T21:10:17.628+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T21:10:17.643+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmprzv_8vc4']
[2024-01-02T21:10:17.637+0000] {standard_task_runner.py:57} INFO - Started process 955 to run task
[2024-01-02T21:10:17.648+0000] {standard_task_runner.py:85} INFO - Job 31: Subtask transform_data
[2024-01-02T21:10:17.828+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host 2590655a4ba7
[2024-01-02T21:10:18.015+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T21:10:18.347+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:46 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T21:10:18.367+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-02T21:10:18.379+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T211017, end_date=20240102T211018
[2024-01-02T21:10:18.466+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-02T21:10:18.504+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-01-02T21:11:39.199+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T21:11:39.212+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T21:11:39.213+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T21:11:39.255+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T21:11:39.263+0000] {standard_task_runner.py:57} INFO - Started process 1031 to run task
[2024-01-02T21:11:39.268+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmpe5lavqzl']
[2024-01-02T21:11:39.271+0000] {standard_task_runner.py:85} INFO - Job 33: Subtask transform_data
[2024-01-02T21:11:39.478+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host 2590655a4ba7
[2024-01-02T21:11:39.710+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T21:11:39.977+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:46 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T21:11:39.994+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-02T21:11:40.007+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T211139, end_date=20240102T211140
[2024-01-02T21:11:40.083+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-02T21:11:40.135+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-01-02T21:32:11.867+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T21:32:11.902+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T21:32:11.903+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T21:32:11.994+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T21:32:12.003+0000] {standard_task_runner.py:57} INFO - Started process 1824 to run task
[2024-01-02T21:32:12.008+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmp2yewys_e']
[2024-01-02T21:32:12.012+0000] {standard_task_runner.py:85} INFO - Job 35: Subtask transform_data
[2024-01-02T21:32:12.102+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host 2590655a4ba7
[2024-01-02T21:32:12.340+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T21:32:12.614+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:46 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T21:32:12.633+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-02T21:32:12.646+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T213211, end_date=20240102T213212
[2024-01-02T21:32:12.744+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-02T21:32:12.782+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-01-02T21:38:08.870+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T21:38:08.883+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T21:38:08.884+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T21:38:08.967+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T21:38:08.974+0000] {standard_task_runner.py:57} INFO - Started process 2075 to run task
[2024-01-02T21:38:08.978+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmp9s8zt56g']
[2024-01-02T21:38:08.981+0000] {standard_task_runner.py:85} INFO - Job 38: Subtask transform_data
[2024-01-02T21:38:09.080+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host 2590655a4ba7
[2024-01-02T21:38:09.259+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T21:38:09.521+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:46 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T21:38:09.537+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-02T21:38:09.552+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T213808, end_date=20240102T213809
[2024-01-02T21:38:09.633+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-02T21:38:09.671+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-01-02T21:44:12.789+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T21:44:12.814+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T21:44:12.815+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T21:44:12.944+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T21:44:12.956+0000] {standard_task_runner.py:57} INFO - Started process 2342 to run task
[2024-01-02T21:44:12.962+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmpsc1td_wx']
[2024-01-02T21:44:12.967+0000] {standard_task_runner.py:85} INFO - Job 41: Subtask transform_data
[2024-01-02T21:44:13.279+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host 2590655a4ba7
[2024-01-02T21:44:13.585+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T21:44:13.899+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:46 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T21:44:13.928+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-02T21:44:13.948+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T214412, end_date=20240102T214413
[2024-01-02T21:44:14.061+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-02T21:44:14.102+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-01-02T22:01:25.700+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T22:01:25.712+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T22:01:25.714+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T22:01:25.787+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T22:01:25.794+0000] {standard_task_runner.py:57} INFO - Started process 3023 to run task
[2024-01-02T22:01:25.799+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmp5sftq2fy']
[2024-01-02T22:01:25.803+0000] {standard_task_runner.py:85} INFO - Job 44: Subtask transform_data
[2024-01-02T22:01:25.892+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host 2590655a4ba7
[2024-01-02T22:01:26.086+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T22:01:26.454+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:46 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T22:01:26.516+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-02T22:01:26.819+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T220125, end_date=20240102T220126
[2024-01-02T22:01:26.907+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-02T22:01:26.955+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-01-02T22:05:30.337+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T22:05:30.368+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T22:05:30.369+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T22:05:30.438+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T22:05:30.445+0000] {standard_task_runner.py:57} INFO - Started process 3192 to run task
[2024-01-02T22:05:30.451+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmpd3plc4en']
[2024-01-02T22:05:30.457+0000] {standard_task_runner.py:85} INFO - Job 47: Subtask transform_data
[2024-01-02T22:05:30.592+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host 2590655a4ba7
[2024-01-02T22:05:30.776+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T22:05:31.208+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:46 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T22:05:31.247+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-02T22:05:31.271+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T220530, end_date=20240102T220531
[2024-01-02T22:05:31.475+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-02T22:05:31.574+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-01-02T22:19:07.615+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T22:19:07.637+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T22:19:07.638+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T22:19:07.677+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T22:19:07.683+0000] {standard_task_runner.py:57} INFO - Started process 3719 to run task
[2024-01-02T22:19:07.687+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmpmxm_j3c1']
[2024-01-02T22:19:07.691+0000] {standard_task_runner.py:85} INFO - Job 49: Subtask transform_data
[2024-01-02T22:19:07.801+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host 2590655a4ba7
[2024-01-02T22:19:08.064+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T22:19:08.310+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:46 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T22:19:08.361+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-02T22:19:08.451+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T221907, end_date=20240102T221908
[2024-01-02T22:19:08.543+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-02T22:19:08.578+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-01-02T22:24:28.564+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T22:24:28.577+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T22:24:28.578+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T22:24:28.620+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T22:24:28.626+0000] {standard_task_runner.py:57} INFO - Started process 3953 to run task
[2024-01-02T22:24:28.630+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmpll4cn9h4']
[2024-01-02T22:24:28.633+0000] {standard_task_runner.py:85} INFO - Job 51: Subtask transform_data
[2024-01-02T22:24:28.746+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host 2590655a4ba7
[2024-01-02T22:24:28.977+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T22:24:29.249+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:46 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T22:24:29.268+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-02T22:24:29.282+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T222428, end_date=20240102T222429
[2024-01-02T22:24:29.406+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-02T22:24:29.445+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-01-02T22:25:43.461+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T22:25:43.477+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T22:25:43.478+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T22:25:43.562+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T22:25:43.570+0000] {standard_task_runner.py:57} INFO - Started process 4021 to run task
[2024-01-02T22:25:43.576+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmp1d99nr9e']
[2024-01-02T22:25:43.581+0000] {standard_task_runner.py:85} INFO - Job 53: Subtask transform_data
[2024-01-02T22:25:43.682+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host 2590655a4ba7
[2024-01-02T22:25:43.834+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T22:25:44.202+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:46 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T22:25:44.220+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-02T22:25:44.236+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T222543, end_date=20240102T222544
[2024-01-02T22:25:44.359+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-02T22:25:44.402+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-01-02T22:42:48.023+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T22:42:48.049+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-02T22:42:48.050+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-02T22:42:48.083+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2024-01-02T22:42:48.091+0000] {standard_task_runner.py:57} INFO - Started process 4709 to run task
[2024-01-02T22:42:48.097+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'booking_ingestion', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_operator2.py', '--cfg-path', '/tmp/tmpql39k2zg']
[2024-01-02T22:42:48.103+0000] {standard_task_runner.py:85} INFO - Job 55: Subtask transform_data
[2024-01-02T22:42:48.229+0000] {task_command.py:415} INFO - Running <TaskInstance: booking_ingestion.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host 2590655a4ba7
[2024-01-02T22:42:48.406+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='booking_ingestion' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-02T22:42:48.695+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/dag_with_postgress_operator2.py:47 UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.
[2024-01-02T22:42:48.720+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-02T22:42:48.737+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=booking_ingestion, task_id=transform_data, execution_date=20240101T000000, start_date=20240102T224248, end_date=20240102T224248
[2024-01-02T22:42:48.835+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-02T22:42:48.876+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
